option optimize_for = LITE_RUNTIME;

message GmCmdRequest
{
 required string cmd = 1;
}
